BraveShared/tr.lproj/BraveShared.strings

```diff
@@ -157,6 +157,9 @@
 /* Sync page title */
 "BraveSync" = "Eşitleme";
 
+/* Sync-Internals screen title (Sync Internals or Sync Debugging is fine). */
+"BraveSyncInternalsTitle" = "Dahili Unsurları Eşitle";
```

Comment:
This is super out of context as translation for a screen title which is used for `Sync Debugging`.
`Senkronizasyon Hata Ayıklama` would give the necessary meaning.

Reply:
heh i had the same problem when translating Polish, `internals` word is not used in this context in PL at all